pkg/virt-controller/services/template.go

```diff
@@ -1547,44 +1546,64 @@ func getIfaceByName(vmi *v1.VirtualMachineInstance, name string) *v1.Interface {
 	return nil
 }
 
-func getCniAnnotations(vmi *v1.VirtualMachineInstance) (cniAnnotations map[string]string, err error) {
-	ifaceListMap := make([]map[string]string, 0)
-	cniAnnotations = make(map[string]string, 0)
+func getMultusCNIAnnotation(vmi *v1.VirtualMachineInstance) (string, error) {
+	var ifaceListMap []map[string]string
+
+	relevantNetworks := filterMultusNonDefaultNetworks(vmi)
```

Comment:
How about "networks" or "multusNetworks" ?

Reply:
network can be pretty much anything, and multusNetworks can be multus non-default networks + multus default network. Both are wrong.

Renamed it to what I think is explicit enough.